web/src/admin/Customers.vue

```diff
@@ -13,7 +13,7 @@
     </button>
   </div>
   <div class="customer-container">
-    <p v-if="fetchError" class="m-4">Could not fetch customers: {{ fetchError }}</p>
+    <p v-if="!!fetchError" class="m-4">Could not fetch customers: {{ fetchError }}</p>
```

Comment:
is this correct? (`!!`)

Reply:
its logical correct, but not needed indeed.